open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -34,7 +34,6 @@
     "* Kernel Density Estimation Plots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def label_encode(df, column_name, encoded_column_name):
</pre><p>The second parameter is the column name to encode.</p><p>The 3rd parameter is the new column name for the encoded result.</p><p> </p><p>Please fix the documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>